infra/util/src/test/java/org/apache/shardingsphere/infra/util/directory/ClasspathResourceDirectoryReaderTest.java

```diff
@@ -45,13 +45,13 @@ void assertReadTest() {
         assertThat(resourceNameList.size(), is(5));
         final String separator = File.separator;
         assertThat(resourceNameList, hasItems("yaml" + separator + "accepted-class.yaml", "yaml" + separator + "customized-obj.yaml", "yaml" + separator + "empty-config.yaml",
-                "yaml" + separator + "shortcuts-fixture.yaml", "yaml/fixture/fixture.yaml"));
+                "yaml" + separator + "shortcuts-fixture.yaml", "yaml" + separator + "fixture" + separator + "fixture.yaml"));
     }
     
     @Test
     void assertReadNestedTest() {
         List<String> resourceNameList = ClasspathResourceDirectoryReader.read("yaml/fixture").collect(Collectors.toList());
         assertThat(resourceNameList.size(), is(1));
-        assertThat(resourceNameList, hasItems("yaml/fixture/fixture.yaml"));
+        assertThat(resourceNameList, hasItems("yaml/fixture" + File.separator + "fixture.yaml"));
```

Comment:
- @iamhucong Hi, I was wondering if getting a URL like `yaml/fixture\\fixture.yaml` on Windows 11 is the expected behavior expected by #30361 ?

Reply:
@linghengqian It should be a path that can obtain resources through the ClassLoader#getResource method. Thank you for your fix.